test.py

```diff
@@ -1263,7 +1280,9 @@ def report_error(error):
                 print(test.shortname, end=" ")
                 report_error("Test was cancelled: the parent process is exiting")
         except Exception as e:
-            report_error("Failed to run the test:\n{e}".format(e=e))
+            with open(f'{options.tmpdir}/stack.out', 'w+') as file:
+                traceback.print_exc(file=file)
```

Comment:
I don't understand the purpose of this change.

Reply:
This is leftovers after debugging. I'll revert this.